ingest/manifest_one_vs_rest.tsv

```diff
@@ -0,0 +1 @@
+type_0	type_1	type_2	type_3

```

Comment:
Blocker: please see [Jon's note](https://github.com/broadinstitute/scp-ingest-pipeline/pull/309#discussion_r1220103112) about not using two manifest files.  Let's use just one.  His example in that link is a good reference of how the single manifest file needs to look, and how to distinguish one-vs-rest and pairwise comparisons.

Reply:
correct manifest file uploaded - wrong one was provided initially, my bad!